cluster/addons/dns/skydns-rc.yaml.base

```diff
@@ -44,7 +44,7 @@ spec:
     spec:
       containers:
       - name: kubedns
-        image: gcr.io/google_containers/kubedns-amd64:1.8
+        image: gcr.io/bowei-gke-dev/kubedns-amd64:testing
```

Comment:
I think we need to have a more formal image release workflow, as to make the e2e tests pass, they need to refer to updated images, however, the images would then exist without a matching change committed in the repository.